modules/memfault-firmware-sdk/memfault_ncs_metrics.c

```diff
@@ -58,150 +69,27 @@ static void stack_check(const struct k_thread *cthread, void *user_data)
 		return;
 	}
 
-	if (stack_type == STACK_CONNECTION_POLL) {
-		LOG_DBG("Unused connection_poll_thread stack size: %d", unused);
+	LOG_DBG("Unused %s stack size: %d", log_strdup(thread_metrics->thread_name), unused);
```

Comment:
Looks like after the latest upmerge it is no longer needed to call `log_strdup` while providing strings to the `LOG_` macros. https://github.com/zephyrproject-rtos/zephyr/commit/041f0e5379

Reply:
Thanks, done